src/js/edu-benefits/1990e/config/form.js

```diff
@@ -190,6 +192,32 @@ const formConfig = {
     employmentHistory: {
       title: 'Employment History',
       pages: {
+        employmentHistory: {
+          title: 'Employment History',
+          path: 'employment-history',
+          uiSchema: {
+            employmentHistory: {
+              'view:hasNonMilitaryJobs': {
+                'ui:title': 'Have you ever held a license of journeyman rating (for example, as a contractor or plumber) to practice a profession?'
+              },
+              nonMilitaryJobs: _.set(['ui:options', 'expandUnder'], 'view:hasNonMilitaryJobs', nonMilitaryJobsUiSchema)
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              employmentHistory: {
+                type: 'object',
+                properties: {
+                  'view:hasNonMilitaryJobs': {
+                    type: 'boolean'
+                  },
+                  nonMilitaryJobs: _.unset('items.properties.postMilitaryJob', nonMilitaryJobs)
```

Comment:
I need to make an item to get this out of the schema for these two forms, I think.